### DISCUSSION TITLE
本项目有准备支持quic吗？

### DISCUSSION THREAD

**Discussion by wpyok500** [SeaHOH/GotoX#40] [点子]
@seaHOH 本项目有准备支持quic吗？

**Comment by SeaHOH** [comment]
如果有现成的模块支持，会尝试下。

**Comment by blob2015** [comment]
大海兄，你的c1、c2啥区别啊

**Comment by SeaHOH** [comment]
写的有啊
>文件名后面的 c + 数字 代表服务端代码之外的改变。

**Comment by ottilis** [comment]
需要quic啊，我这边昨天傍晚后，一个ip都没了，昨晚扫一晚上也没。quic还有。

**Comment by SeaHOH** [comment]
Python 下我无能为力，没有现成模块可用。

**Comment by SeaHOH** [comment]
https://github.com/myguybetter/GoogleIPHunter

**Comment by ottilis** [comment]
好，下了那个。

**Comment by ghost** [comment]
Python是解释性语言, 底层就是用c实现的, 所以用python调用C是很容易的
http://blog.csdn.net/fxjtoday/article/details/6059874
现成的 Python wrapper for QUIC 
https://github.com/osdf/pyquic
所以支持quic不难吧

**Comment by SeaHOH** [comment]
其实是我没有编译工具……

没有现成库文件是因为还没稳定下来，不然 PyPI 会有发布的。

**Comment by k79e** [comment]
go好像有quic了?
https://godoc.org/github.com/lucas-clemente/quic-go

**Comment by SeaHOH** [comment]
已经有纯 py 的了，由于各方面的原因暂时还没打算上。
